README.md

```
@@ -1 +1,60 @@
-[link to demo](https://nu-on-web.github.io/nu-on-web/)
+# nu-on-web
+
+A WebAssembly port of [NuShell](https://www.nushell.sh), enabling you to run NuShell pipelines directly in your browser.
+
+## Demo
+
+Explore the live demo here: https://nu-on-web.github.io/nu-on-web/
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)